fiftyone/server/routes/embeddings_v2.py

```
@@ -0,0 +1,755 @@
+    dataset = fosu.load_and_cache_dataset(dataset_name)
+    results = dataset.load_brain_results(brain_key)
+    if results is None:
+        raise ValueError(
+            f"Results for brain run with key '{brain_key}' are not yet "
+            "available"
+        )
+
+    return dataset, results
+
+
+def _slice(array, data):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_slice`

(ANN202)